target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom

```diff
@@ -97,7 +97,8 @@ case "$FIRMWARE" in
 	tplink,tl-wdr4300-v1-il|\
 	tplink,tl-wdr4310-v1|\
 	tplink,tl-wdr4900-v2|\
-	winchannel,wb2000)
+	winchannel,wb2000|\
+	mercury,mw4530r-v1)
```

Comment:
The node should be sorted alphabetically internally, so mercury as the first member, and then the whole node should be moved based on sorting of the first entry.